test/fixtures/convert/decorator/decorator-on-class-with-argument/flow.js

```diff
@@ -0,0 +1,12 @@
+import { autoBindMethodsForReact } from "class-autobind-decorator";
+
+@autoBindMethodsForReact(AUTOBIND_CFG)
+class PasswordEditor extends React.PureComponent<Props, State> {
```

Comment:
It seems like the `decorator-factory` example already covers this, unless I'm missing something.

Reply:
I can remove it if you prefer fewer tests, but usually, more tests are better 😄

Reply:
It's fine.  It'd be a different story if the tests took a long time to run, but that isn't the case.